metabrainz/static/css/main.less

```diff
@@ -211,6 +211,26 @@ h1.page-title, h2.page-title {
   }
 }
 
+//team page 
+#team {
+  strong { color: @green; }
+  .people { border-bottom: 2px solid @green; }
```

Comment:
Oddly `hr` is defined in HTML5 as a "thematic break" that doesn't necessarily have a visual component, losing all of it's attributes. ([Source](http://www.w3schools.com/tags/tag_hr.asp))